client-library/src/Attestation/AttestationClient/lib/DynamicLibrary/debian/changelog

```diff
@@ -0,0 +1,7 @@
+azguestattestation1 (1.0.2) stable; urgency=medium
+  * Initial release of azguestattestation library.
+
+ -- Vinay Kotak <vikotak@microsoft.com>  Thu, 22 July 2021 17:45:00 -0700
```

Comment:
are these needed?

Reply:
changelog file is needed for building debian package

Reply:
removed as suggested